src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx

```diff
@@ -186,6 +194,7 @@ export const mapStateToProps = (state) => ({
   transcripts: selectors.video.transcripts(state),
   allowTranscriptDownloads: selectors.video.allowTranscriptDownloads(state),
   showTranscriptByDefault: selectors.video.showTranscriptByDefault(state),
+  transcriptCheckCommand: selectors.video.transcriptCheckCommand(state),
```

Comment:
instead of using a variable stored in the video state here as a variable, it would be simpler to just query the request state of the transcriptcheck call ('selectors.requests.isCompleted(transcriptCheck)' or something like that). It will reduce the complexity of the video redux state.

Reply:
I need the selector because that has a bool that determines if there are transcripts available to import. So I am not relying on the completion of the API call to determine whether to show the card or not.

Reply:
ah gotcha -- late afternoon brain IDk why it got into my head that that was the feature lol